config/scripts/validate-intl-keys.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+yarn build-intl-keys
+
+if (git diff --quiet i18n/keys.json > /dev/null); then
+  echo "i18n/keys.json is up to date."
```

Comment:
isn't this then/else inverted? If we get into the `then` it should mean we're out of date, right?

Reply:
it works for me

Reply:
yes it's works, it's just bash exit code that's misleading in the condition